Makefile

```diff
@@ -50,11 +50,11 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.36.1
+OPERATOR_SDK_VERSION ?= v1.38.0
```

Comment:
It is fine to skip v1.37.0 if it doesn't affect us, which seems to be the case from https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.37.0/, but it should be mentioned explicitly in the commit message for extra clarity.

Reply:
Thanks! I forget that.